libs/effects-ng/src/lib/effects-ng.module.ts

```diff
@@ -7,15 +7,14 @@ import {
 } from '@angular/core';
 import {
   actions,
-  Effect,
   EffectsConfig,
   EffectsManager,
   initEffects,
-  isEffect,
   registerEffects,
 } from '@ngneat/effects';
 import { Actions } from './actions';
 import { EFFECTS_MANAGER, EFFECTS_PROVIDERS } from './tokens';
+import { retrieveOnlyEffects } from './utils';
 
 @NgModule()
 export class EffectsNgModule {
```

Comment:
We don't want to remove the module?

Reply:
We can deprecate it and leave it for a while until the standalone API is wideused.

Reply:
Sounds good